# -*- coding: utf-8 -*-

from PyQt4 import QtCore, QtGui

try:
    _fromUtf8 = QtCore.QString.fromUtf8
except AttributeError:
    def _fromUtf8(s):
        return s

try:
    _encoding = QtGui.QApplication.UnicodeUTF8
    def _translate(context, text, disambig):
        return QtGui.QApplication.translate(context, text, disambig, _encoding)
except AttributeError:
    def _translate(context, text, disambig):
        return QtGui.QApplication.translate(context, text, disambig)


class Ui_VisualizarFornecedor(object):
    def setupUi(self, VisualizarFornecedor):
        VisualizarFornecedor.setObjectName(_fromUtf8("VisualizarFornecedor"))
        VisualizarFornecedor.resize(1324, 537)
        VisualizarFornecedor.move(400, 250)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap(_fromUtf8("../Icons/mdpi.png")), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        VisualizarFornecedor.setWindowIcon(icon)
        self.tblFornecedor = QtGui.QTableWidget(VisualizarFornecedor)
        self.tblFornecedor.setGeometry(QtCore.QRect(0, 0, 931, 535))
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Preferred, QtGui.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.tblFornecedor.sizePolicy().hasHeightForWidth())
        self.tblFornecedor.setSizePolicy(sizePolicy)
        self.tblFornecedor.setHorizontalScrollBarPolicy(QtCore.Qt.ScrollBarAlwaysOff)
        self.tblFornecedor.setSelectionBehavior(QtGui.QAbstractItemView.SelectRows)
        self.tblFornecedor.setObjectName(_fromUtf8("tblFornecedor"))
        self.tblFornecedor.setColumnCount(7)
        self.tblFornecedor.setRowCount(0)
        item = QtGui.QTableWidgetItem()
        self.tblFornecedor.setHorizontalHeaderItem(0, item)
        item = QtGui.QTableWidgetItem()
        self.tblFornecedor.setHorizontalHeaderItem(1, item)
        item = QtGui.QTableWidgetItem()
        self.tblFornecedor.setHorizontalHeaderItem(2, item)
        item = QtGui.QTableWidgetItem()
        self.tblFornecedor.setHorizontalHeaderItem(3, item)
        item = QtGui.QTableWidgetItem()
        self.tblFornecedor.setHorizontalHeaderItem(4, item)
        item = QtGui.QTableWidgetItem()
        self.tblFornecedor.setHorizontalHeaderItem(5, item)
        item = QtGui.QTableWidgetItem()
        self.tblFornecedor.setHorizontalHeaderItem(6, item)
        self.tblFornecedor.horizontalHeader().setVisible(True)
        self.tblFornecedor.horizontalHeader().setDefaultSectionSize(133)
        self.tblFornecedor.verticalHeader().setVisible(False)
        self.tblTelefone = QtGui.QTableWidget(VisualizarFornecedor)
        self.tblTelefone.setGeometry(QtCore.QRect(940, 0, 381, 264))
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Maximum, QtGui.QSizePolicy.Maximum)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.tblTelefone.sizePolicy().hasHeightForWidth())
        self.tblTelefone.setSizePolicy(sizePolicy)
        self.tblTelefone.setMaximumSize(QtCore.QSize(541, 264))
        self.tblTelefone.setAutoFillBackground(False)
        self.tblTelefone.setFrameShape(QtGui.QFrame.StyledPanel)
        self.tblTelefone.setFrameShadow(QtGui.QFrame.Sunken)
        self.tblTelefone.setLineWidth(1)
        self.tblTelefone.setHorizontalScrollBarPolicy(QtCore.Qt.ScrollBarAlwaysOff)
        self.tblTelefone.setEditTriggers(QtGui.QAbstractItemView.NoEditTriggers)
        self.tblTelefone.setSelectionBehavior(QtGui.QAbstractItemView.SelectRows)
        self.tblTelefone.setShowGrid(True)
        self.tblTelefone.setGridStyle(QtCore.Qt.SolidLine)
        self.tblTelefone.setWordWrap(True)
        self.tblTelefone.setCornerButtonEnabled(True)
        self.tblTelefone.setObjectName(_fromUtf8("tblTelefone"))
        self.tblTelefone.setColumnCount(3)
        self.tblTelefone.setRowCount(0)
        item = QtGui.QTableWidgetItem()
        self.tblTelefone.setHorizontalHeaderItem(0, item)
        item = QtGui.QTableWidgetItem()
        self.tblTelefone.setHorizontalHeaderItem(1, item)
        item = QtGui.QTableWidgetItem()
        self.tblTelefone.setHorizontalHeaderItem(2, item)
        self.tblTelefone.horizontalHeader().setVisible(True)
        self.tblTelefone.horizontalHeader().setCascadingSectionResizes(False)
        self.tblTelefone.horizontalHeader().setDefaultSectionSize(126)
        self.tblTelefone.horizontalHeader().setSortIndicatorShown(False)
        self.tblTelefone.horizontalHeader().setStretchLastSection(False)
        self.tblTelefone.verticalHeader().setVisible(False)
        self.tblTelefone.verticalHeader().setHighlightSections(True)
        self.tblEmail = QtGui.QTableWidget(VisualizarFornecedor)
        self.tblEmail.setGeometry(QtCore.QRect(940, 270, 381, 264))
        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Maximum, QtGui.QSizePolicy.Maximum)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.tblEmail.sizePolicy().hasHeightForWidth())
        self.tblEmail.setSizePolicy(sizePolicy)
        self.tblEmail.setMaximumSize(QtCore.QSize(541, 264))
        self.tblEmail.setAutoFillBackground(False)
        self.tblEmail.setFrameShape(QtGui.QFrame.StyledPanel)
        self.tblEmail.setFrameShadow(QtGui.QFrame.Sunken)
        self.tblEmail.setLineWidth(1)
        self.tblEmail.setHorizontalScrollBarPolicy(QtCore.Qt.ScrollBarAlwaysOff)
        self.tblEmail.setEditTriggers(QtGui.QAbstractItemView.NoEditTriggers)
        self.tblEmail.setSelectionBehavior(QtGui.QAbstractItemView.SelectRows)
        self.tblEmail.setShowGrid(True)
        self.tblEmail.setGridStyle(QtCore.Qt.SolidLine)
        self.tblEmail.setWordWrap(True)
        self.tblEmail.setCornerButtonEnabled(True)
        self.tblEmail.setObjectName(_fromUtf8("tblEmail"))
        self.tblEmail.setColumnCount(2)
        self.tblEmail.setRowCount(0)
        item = QtGui.QTableWidgetItem()
        self.tblEmail.setHorizontalHeaderItem(0, item)
        item = QtGui.QTableWidgetItem()
        self.tblEmail.setHorizontalHeaderItem(1, item)
        self.tblEmail.horizontalHeader().setVisible(True)
        self.tblEmail.horizontalHeader().setCascadingSectionResizes(False)
        self.tblEmail.horizontalHeader().setDefaultSectionSize(190)
        self.tblEmail.horizontalHeader().setSortIndicatorShown(False)
        self.tblEmail.horizontalHeader().setStretchLastSection(False)
        self.tblEmail.verticalHeader().setVisible(False)
        self.tblEmail.verticalHeader().setHighlightSections(True)

        self.retranslateUi(VisualizarFornecedor)
        QtCore.QMetaObject.connectSlotsByName(VisualizarFornecedor)

    def retranslateUi(self, VisualizarFornecedor):
        VisualizarFornecedor.setWindowTitle(_translate("VisualizarFornecedor", "Visualizar Fornecedor", None))
        item = self.tblFornecedor.horizontalHeaderItem(0)
        item.setText(_translate("VisualizarFornecedor", "Id", None))
        item = self.tblFornecedor.horizontalHeaderItem(1)
        item.setText(_translate("VisualizarFornecedor", "Nome", None))
        item = self.tblFornecedor.horizontalHeaderItem(2)
        item.setText(_translate("VisualizarFornecedor", "Descrição", None))
        item = self.tblFornecedor.horizontalHeaderItem(3)
        item.setText(_translate("VisualizarFornecedor", "Cidade", None))
        item = self.tblFornecedor.horizontalHeaderItem(4)
        item.setText(_translate("VisualizarFornecedor", "Endereço", None))
        item = self.tblFornecedor.horizontalHeaderItem(5)
        item.setText(_translate("VisualizarFornecedor", "Bairro", None))
        item = self.tblFornecedor.horizontalHeaderItem(6)
        item.setText(_translate("VisualizarFornecedor", "Numero", None))
        self.tblTelefone.setSortingEnabled(False)
        item = self.tblTelefone.horizontalHeaderItem(0)
        item.setText(_translate("VisualizarFornecedor", "DDD", None))
        item = self.tblTelefone.horizontalHeaderItem(1)
        item.setText(_translate("VisualizarFornecedor", "Numero", None))
        item = self.tblTelefone.horizontalHeaderItem(2)
        item.setText(_translate("VisualizarFornecedor", "Referencia", None))
        self.tblEmail.setSortingEnabled(False)
        item = self.tblEmail.horizontalHeaderItem(0)
        item.setText(_translate("VisualizarFornecedor", "Email", None))
        item = self.tblEmail.horizontalHeaderItem(1)
        item.setText(_translate("VisualizarFornecedor", "Referencia", None))

